### PR TITLE
python-makeelf: Added package python-makeelf

### PR DESCRIPTION
**Pull request by AdamJovanovic**
Maintainer: me
Compile tested: BCM2711 for R-Pi. Compiled at OpenWrt v22.03.4
Run tested: BCM2711 for R-Pi. Tested at OpenWrt v22.03.4

Description:
Added the Python package makeelf. Tested on a Raspberry Pi by running a script which has python-makeelf as a dependency.

### CODE REVIEW

**Comment by BKPepe** [comment]
Which script requires this package? Is there any package which would depend on this package in the future?

**Comment by AdamJovanovic** [comment]
@BKPepe I can't say I've seen any examples where it would be a dependency. I am proposing it to be added as I am using it to parse binary files, and I believe this could be useful for others that may need to do this in the future.

File: lang/python/python-makeelf/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-makeelf
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=1
+
+PYPI_NAME:=makeelf
+PKG_HASH:=9d8b8cf16b00116ea72d96f9f9561c0f960bedc78e68743e2459f1f1d71e0572
+
+PKG_MAINTAINER:=Adam Jovanovic <<adamjovanovix@gmail.com>>
```
Comment (jefferyto) [inline lang/python/python-makeelf/Makefile:10]:
```suggestion
PKG_MAINTAINER:=Adam Jovanovic <adamjovanovix@gmail.com>
```

File: lang/python/python-makeelf/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-makeelf
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=A Python library for parsing and creating ELFs
+  URL:=https://github.com/v3l0c1r4pt0r/makeelf
+endef
```
Comment (jefferyto) [inline lang/python/python-makeelf/Makefile:26]: Missing `DEPENDS`.

Suggest you run `make package/python-makeelf/configure PY3=stdlib V=s` to find any standard library dependencies (see https://github.com/openwrt/packages/tree/master/lang/python#python-package-dependencies).

**Review by jefferyto — COMMENTED** [review]
Also:

* The email address you have configured in your git client does not match the email address in the `Signed-off-by` line (this is why the [Test Formalities](https://github.com/openwrt/packages/actions/runs/4685869733/jobs/8320696000?pr=20796) failed)
* Suggest to squash the commits

**Comment by BKPepe** [comment]
What's happening with this PR? @AdamJovanovic will you still work on this?

**Comment by BKPepe** [comment]
Closing.

**Pull request closed without merging**